app/css/buddyup.css

```diff
@@ -116,6 +116,15 @@ html {
     padding: .7rem .9rem;
     border-radius: 6px;
 }
+.overlay {
```

Comment:
Let's move this to utils.css. We should not add anything to buddyup.css because it is a mix of many pages. I don't like utils.css a lot but it's still makes more sense there.